shared/types/genesis/proto/config.proto

```diff
@@ -54,4 +55,11 @@ message TelemetryConfig {// TODO (team) move to shared/telemetry
   bool enabled = 1;
   string address = 2; // The address the telemetry module will use to listen for metrics PULL requests (e.g. 0.0.0.0:9000 for prometheus)
   string endpoint = 3; // The endpoint available to fetch recorded metrics (e.g. /metrics for prometheus)
+}
+
+message RPCConfig { // DISCUSS (team): with #163 in mind, where should this reside?
```

Comment:
Let's discuss #163 as a team next week, because I think this approach speaks well to how easy it is to additional configs.

cc @andrewnguyen22 for context